docs/recipes/browserify.md

```diff
@@ -24,25 +24,26 @@ $ npm install --save-dev browserify babelify vinyl-buffer vinyl-source-stream
 ```
 
 ```diff
-gulp.task('scripts', () => {
--  return gulp.src('app/scripts/**/*.js')
-+
+function scripts() {
+-  return src('app/scripts/**/*.js')
 +  const b = browserify({
 +    entries: 'app/scripts/main.js',
-+    transform: babelify,
 +    debug: true
-+  });
++  })
++  .transform(babelify.configure({
++    presets: ["@babel/preset-env"]
```

Comment:
Since generator-webapp comes with `.babelrc` which contains `@babel/preset-env`, is configuring babelify here necessary? Won't it just pick up config from `.babelrc`?

Reply:
You are completely right, even if set other babelify options leaving out the preset it will still pick up the config from  `.babelrc`.